server/src/main/java/org/spine3/server/BoundedContext.java

```diff
@@ -316,39 +292,55 @@ public void notify(IntegrationEvent integrationEvent,
     }
 
     /** Obtains instance of {@link CommandBus} of this {@code BoundedContext}. */
-    @CheckReturnValue
     public CommandBus getCommandBus() {
         return this.commandBus;
     }
 
     /** Obtains instance of {@link EventBus} of this {@code BoundedContext}. */
-    @CheckReturnValue
     public EventBus getEventBus() {
         return this.eventBus;
     }
 
     /** Obtains instance of {@link FailureBus} of this {@code BoundedContext}. */
-    @CheckReturnValue
     public FailureBus getFailureBus() {
         return this.commandBus.failureBus();
     }
 
     /** Obtains instance of {@link Stand} of this {@code BoundedContext}. */
-    @CheckReturnValue
     public Stand getStand() {
         return stand;
     }
 
     /**
+     * Obtains a set of entity type names by their visibility.
+     */
+    public Set<TypeName> getEntityTypes(Visibility visibility) {
+        final Set<TypeName> result = guard.getEntityTypes(visibility);
+        return result;
+    }
+
+    /**
      * Obtains an {@code AggregateRepository} which manages aggregates with the passed state.
      *
      * @param aggregateStateClass the class of the aggregate state
      * @return repository instance or empty {@code Optional} if not found
      */
     public Optional<? extends AggregateRepository<?, ?>> getAggregateRepository(
             Class<? extends Message> aggregateStateClass) {
-        final AggregateRepository<?, ?> result = aggregateRepositories.get(aggregateStateClass);
-        return Optional.fromNullable(result);
+        // See if there is a repository for this state at all.
+        if (!guard.hasRepository(aggregateStateClass)) {
+            throw newIllegalStateException("No repository found for the aggregate state class %s",
```

Comment:
Seems to be a bug of Codecov. There's a test for it. Please see `BoundedContextShould.throw_ISE_when_no_repository_registered()`.